nautobot_golden_config/migrations/0026_configplan.py

```diff
@@ -43,12 +44,14 @@ class Migration(migrations.Migration):
                 ),
                 (
                     "feature",
+                    models.ManyToManyField(
```

Comment:
I think we should re-think this idea.
There's a few things we should carefully think about : 
- ordering of features 
- priorities of features
- filtering of config plans

I think same effect could be also achieved today by doing manual config plan and navigating like this

`device.compliance_result.remediation` 
`device.compliance_result.missing` etc

Reply:
What is the difference between ordering and priority?

Reply:
I meant those two are important in this sense:
- I want my `^username` feature-section before `^aaa` feature-section in my ConfigPlan
- I want my `username` feature to be pushed to the device before my `aaa` feature

So there could be:
- feature to configPlan merge ordering
- config plan deploy ordering